.craft.yml

```diff
@@ -6,7 +6,7 @@ statusProvider:
   name: github
   config:
     contexts:
-      - 'onpremise-builder (sentryio)'
+      - 'self-hosted-builder (sentryio)'
```

Comment:
https://github.com/getsentry/craft#status-provider

Reply:
```ts
        const contextResult = this.getStatusForContext(
          contextString,
          revisionStatus,
          revisionCheckSuites,
          revisionChecks
        );
```

https://github.com/getsentry/craft/blob/1e352b9e6fd486abc16f7f249b243929b26dd679/src/status_providers/github.ts#L69-L74

Reply:
E.g. https://github.com/getsentry/sentry/runs/4448300537

Where is it on this PR? 🤔

Reply:
Is this where I have to do /gcbrun? 🧐

Reply:
Alright yeah this is [where I start to fall off](https://develop.sentry.dev/continuous-integration/#docker-images) in terms of permissions/visibility ... I gather that there is some manually configured job name in GCR ... 🤨

Reply:
> where I start to fall off in terms of permissions/visibility

OR DO I!?!?!?!?!?!

<img width="549" alt="Screen Shot 2021-12-07 at 5 25 27 PM" src="https://user-images.githubusercontent.com/134455/145115492-0bc07439-a63e-4b1f-8fe9-480055032832.png">

💃

Reply:
Of course, Google may very well show me a "Save" button but throw an error if I click it. 🙄

Reply:
Nope, I can edit. 😁

<img width="537" alt="Screen Shot 2021-12-07 at 5 29 50 PM" src="https://user-images.githubusercontent.com/134455/145115882-e468e4de-520c-471d-8046-6cd04f5b104b.png">

Reply:
Okay! Sooooooooo ... when this PR lands I intend to edit the two triggers in GCB. I'll keep an eye on this during the 21.12.0 release ... kicked out https://github.com/getsentry/self-hosted/issues/1193 for that.